packages/sol-cov/test/instructions_test.ts

```diff
@@ -0,0 +1,22 @@
+import * as chai from 'chai';
+import * as fs from 'fs';
+import 'mocha';
+import * as path from 'path';
+
+import { getPcToInstructionIndexMapping } from '../src/instructions';
+
+const expect = chai.expect;
+
+describe('instructions', () => {
+    describe('#getPcToInstructionIndexMapping', () => {
+        it('correctly maps pcs to instruction indexed', () => {
+            const PUSH1 = 0x60;
+            const PUSH2 = 0x61;
+            const TIMESTAMP = 0x42;
```

Comment:
There are now repeated in two places in the codebase. Should we move them to `constants.ts`?